src/platforms/android/common/performance/instrumentation/automatic-instrumentation.mdx

```diff
@@ -110,6 +110,18 @@ fun displayUserData() {
 
 When you didn't finish the transaction yet, but you start a new Activity, the SDK always automatically finishes the previous Activity transaction. This is due to the fact that only one transaction at a time can be bound to the Scope. To work around that, you may create transactions manually using the custom instrumentation and its instance to start spans instead of the Static API.
 
+### Android's Fragment's Instrumentation
+
+The Fragment's instrumentation, once enabled, starts a span for each launch of a Fragment. The SDK sets the Span operation to `ui.fragment.load` and the Span description to the Fragment's name, for example, `LoginFragment`.
```

Comment:
Android docs don't capitalize "fragment" after the first use of it (where it's a linked definition), so I've taken out the capital letter except in places where you use the full title again. I've also done the same with the word "span." 

```suggestion
Once enabled, Android's Fragment Instrumentation starts a span for each launch of a fragment. The SDK sets the span operation to `ui.fragment.load` and the span description to the fragment's name, for example, `LoginFragment`.
```